test/integration/removeUnwantedIndexes.js

```diff
@@ -0,0 +1,120 @@
+const chai = require('chai');
+const moment = require('moment');
+
+const esClient = require('../../lib/es/esClient');
+const removeUnwantedIndexes = require('../../lib/es/removeUnwantedIndexes');
+const sampleData = require('../resources/sample-data');
+
+const expect = chai.expect;
+const maxWaitTime = 1 * 60 * 1000;
```

Comment:
Does this need to include the multiplication by `1`?

Reply:
No, I imagine it used to be more than one minute, I'll remove the 1 for clarity